src/util/localStorageActions.ts

```diff
@@ -0,0 +1,21 @@
+export const getFromStorage = (key: string) => {
+  const storedData: string | null = localStorage.getItem(key);
+
+  if (storedData) return JSON.parse(storedData);
+
+  return null;
```

Comment:
No need to return null JSON.parse(null) is null.
```suggestion
  const storedData: string | null = localStorage.getItem(key);
  return JSON.parse(storedData);
```